### PR TITLE
Fix: 보안 상의 문제로 리프레시 토큰은 HTTP-Only 쿠키를 통해 전달하도록 수정함

### LINKED ISSUE #75: [FEAT] 보안 상의 이유로 응답 페이로드 대신 HTTP-Only 쿠키를 통해 리프레시 토큰을 전달하도록 수정함
### 📝 기능 설명

보안 상의 이유로 응답 페이로드 대신 HTTP-Only 쿠키를 통해 리프레시 토큰을 전달하도록 수정합니다.

### 작업 내용

- [x] 응답 페이로드 본문 대신 쿠키를 통해서 리프레시 토큰을 전달하도록 수정함
- [x] 미뤄뒀던 인증 컨트롤러 및 서비스 계층에 대한 테스트를 진행함
- [x] 미뤄뒀던 인증 API에 대한 문서화를 진행함

### ⏳ 추정 시간

5시간

### ⌛ 소요 시간

5시간

### PR DESCRIPTION
**Pull request by su6net**
## ✨ 작업 내용
- 보안 상의 문제로 리프레시 토큰은 HTTP-Only 쿠키를 통해 전달하도록 수정했습니다.
- 인증 API에 대한 문서화를 진행했습니다.
- 인증 컨트롤러와 서비스 계층에 대한 테스트 케이스들을 작성했습니다.

## 🔗 관련 이슈
- close #75

### CODE REVIEW

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+class AuthenticationApiV1ControllerTest extends CommonControllerSliceTest {
+
+    @Autowired
+    private JwtLoginService loginService;
+
+    @Autowired
+    private JwtTokenService tokenService;
+
+    @Nested
+    class 사용자가_로그인_시에 {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.TypeNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Type name '사용자가_로그인_시에' must match pattern '^[A-Z][a-zA-Z0-9]*$'.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+        @Test
+        void 성공한다() throws Exception {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '성공한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            mockMvc.perform(post("/api/v1/auth/login")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(request)))
+                    .andExpect(status().isOk())
+                    .andExpect(jsonPath("$.access_token").value(TokenDtoFixture.VALID_ACCESS_TOKEN))
+                    .andExpect(cookie().value("refresh_token", TokenDtoFixture.VALID_REFRESH_TOKEN));
+        }
+
+        @Test
+        void 매치되는_정보가_없으면_예외가_발생한다() throws Exception {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:60]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '매치되는_정보가_없으면_예외가_발생한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(request)))
+                    .andExpect(status().isBadRequest())
+                    .andExpect(jsonPath("$.access_token").doesNotExist())
+                    .andExpect(cookie().doesNotExist("refresh_token"));
+        }
+    }
+
+    @Nested
+    class 사용자가_리프레시_토큰_재발급_시에 {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.TypeNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Type name '사용자가_리프레시_토큰_재발급_시에' must match pattern '^[A-Z][a-zA-Z0-9]*$'.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+        @Test
+        @WithMockUser
+        void 성공한다() throws Exception {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:81]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '성공한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .cookie(new Cookie("refresh_token", TokenDtoFixture.VALID_REFRESH_TOKEN)))
+                    .andExpect(status().isOk())
+                    .andExpect(jsonPath("$.access_token").value(newTokens.accessToken()))
+                    .andExpect(cookie().value("refresh_token", newTokens.refreshToken()));
+        }
+
+        @Test
+        @WithMockUser
+        void 유효하지_않으면_예외가_발생한다() throws Exception {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:98]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '유효하지_않으면_예외가_발생한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+            mockMvc.perform(post("/api/v1/auth/refresh")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .cookie(new Cookie("refresh_token", TokenDtoFixture.INVALID_REFRESH_TOKEN)))
+                    .andExpect(status().isUnauthorized())
+                    .andExpect(jsonPath("$.access_token").doesNotExist());
+        }
+    }
+
+    @Nested
+    class 사용자가_회원가입_시에 {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:112]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.TypeNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Type name '사용자가_회원가입_시에' must match pattern '^[A-Z][a-zA-Z0-9]*$'.

File: src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+    private JwtTokenProvider tokenProvider;
+
+    @Mock
+    private MemberRepository memberRepository;
+
+    @InjectMocks
+    private RefreshTokenService refreshTokenService;
+
+    @Test
+    void 아이디로_리프레시_토큰을_제거한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '아이디로_리프레시_토큰을_제거한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+        // when
+        refreshTokenService.deleteTokenByUsername(member.getUsername());
+
+        // then
+        verify(refreshTokenRepository).deleteByMember(member);
+        verify(refreshTokenRepository).flush();
+    }
+
+    @Test
+    void 리프레시_토큰을_저장한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '리프레시_토큰을_저장한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+        Long actualId = refreshTokenService.save(member.getUsername(), validRefreshToken.getToken());
+
+        // then
+        assertThat(actualId).isEqualTo(expectedId);
+
+        verify(refreshTokenRepository).save(any(RefreshToken.class));
+    }
+
+    @Test
+    void 토큰이_유효하지_않으면_검증에_실패한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:80]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '토큰이_유효하지_않으면_검증에_실패한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+        boolean actualResult = refreshTokenService.validateToken(invalidRefreshToken);
+
+        // then
+        assertThat(actualResult).isEqualTo(expectedResult);
+
+        verifyNoMoreInteractions(refreshTokenRepository);
+    }
+
+    @Test
+    void 유저가_존재하지_않는_경우_예외가_발생한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:97]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '유저가_존재하지_않는_경우_예외가_발생한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+        when(tokenProvider.getUsernameFromToken(validRefreshToken)).thenReturn(nonExistentUsername);
+        doThrow(new EntityNotFoundException(ErrorCode.MEMBER_NOT_EXIST)).when(memberRepository).findByUsernameOrThrow(nonExistentUsername);
+
+        // when & then
+        assertThatThrownBy(() -> refreshTokenService.validateToken(validRefreshToken))
+                .isInstanceOf(EntityNotFoundException.class);
+    }
+
+    @Test
+    void DB에_리프레시_토큰이_존재하지_않는_경우_검증에_실패한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:112]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'DB에_리프레시_토큰이_존재하지_않는_경우_검증에_실패한다' must contain no more than '1' consecutive capital letters.

Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:112]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name 'DB에_리프레시_토큰이_존재하지_않는_경우_검증에_실패한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+
+        // when
+        boolean actualResult = refreshTokenService.validateToken(validRefreshToken);
+
+        // then
+        assertThat(actualResult).isEqualTo(expectedResult);
+    }
+
+    @Test
+    void 만료된_토큰이면_참을_반환한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:131]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '만료된_토큰이면_참을_반환한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+
+        // when
+        boolean actualResult = refreshTokenService.isTokenExpired(expiredRefreshToken);
+
+        // then
+        assertThat(actualResult).isEqualTo(expectedResult);
+    }
+
+    @Test
+    void 만료되지_않은_토큰이면_거짓을_반환한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:144]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '만료되지_않은_토큰이면_거짓을_반환한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/JwtTokenServiceTest.java [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+    private JwtTokenProvider tokenProvider;
+
+    @Mock
+    private RefreshTokenService refreshTokenService;
+
+    @InjectMocks
+    private JwtTokenService tokenService;
+
+    @Test
+    void 리프레시_토큰이_유효하면_새_리프레시_토큰을_생성한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/JwtTokenServiceTest.java:42]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '리프레시_토큰이_유효하면_새_리프레시_토큰을_생성한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/JwtTokenServiceTest.java [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+
+        // then
+        assertThat(actualTokenDto).isEqualTo(expectedTokenDto);
+
+        verify(refreshTokenService).deleteTokenByUsername(username);
+        verify(refreshTokenService).save(username, expectedTokenDto.refreshToken());
+    }
+
+    @Test
+    void 리프레시_토큰이_유효하지_않으면_예외가_발생한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/JwtTokenServiceTest.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '리프레시_토큰이_유효하지_않으면_예외가_발생한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/JwtTokenServiceTest.java [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+
+        doThrow(new AuthenticationException(ErrorCode.INVALID_TOKEN)).when(refreshTokenService).validateToken(invalidRefreshToken);
+
+        // when & then
+        assertThatThrownBy(() -> tokenService.refreshJwtTokens(invalidRefreshToken))
+                .isInstanceOf(AuthenticationException.class);
+    }
+
+    @Test
+    void 토큰을_새롭게_생성한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/JwtTokenServiceTest.java:79]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '토큰을_새롭게_생성한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/JwtLoginServiceTest.java [hunk 18]
```diff
@@ -0,0 +1,85 @@
...
+    @InjectMocks
+    private JwtLoginService loginService;
+
+    @BeforeEach
+    void setUp() {
+        when(authenticationManagerBuilder.getObject()).thenReturn(authenticationManager);
+    }
+
+    @Test
+    void 인증에_성공한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/JwtLoginServiceTest.java:49]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '인증에_성공한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/JwtLoginServiceTest.java [hunk 19]
```diff
@@ -0,0 +1,85 @@
...
+        assertThat(actualTokenDto).isNotNull();
+        assertThat(expectedTokenDto.accessToken()).isEqualTo(actualTokenDto.accessToken());
+        assertThat(expectedTokenDto.refreshToken()).isEqualTo(actualTokenDto.refreshToken());
+
+        verify(authenticationManager).authenticate(token);
+        verify(tokenService).generateJwtTokens(request.getUsername(), authentication);
+    }
+
+    @Test
+    void 인증에_실패하면_예외가_발생한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/JwtLoginServiceTest.java:72]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '인증에_실패하면_예외가_발생한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/LoginServiceTest.java [hunk 20]
```diff
@@ -0,0 +1,57 @@
...
+class LoginServiceTest {
+
+    @Mock
+    private MemberRepository memberRepository;
+
+    @InjectMocks
+    private LoginService loginService;
+
+    @Test
+    void 유저가_존재하면_성공한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/LoginServiceTest.java:32]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '유저가_존재하면_성공한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/auth/service/LoginServiceTest.java [hunk 21]
```diff
@@ -0,0 +1,57 @@
...
+        // when
+        UserDetails userDetails = loginService.loadUserByUsername(validMember.getUsername());
+
+        // then
+        assertThat(userDetails.getUsername()).isEqualTo(validMember.getUsername());
+        assertThat(userDetails.getPassword()).isEqualTo(validMember.getPassword());
+    }
+
+    @Test
+    void 유저가_존재하지_않으면_예외가_발생한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/LoginServiceTest.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '유저가_존재하지_않으면_예외가_발생한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/common/validation/FieldMatchValidatorTest.java [hunk 22]
```diff
@@ -27,10 +31,9 @@ void setUp() {
     }
 
     @Test
-    @DisplayName("서로 일치하면 유효성 검사가 성공한다.")
-    void success() {
+    void 서로_일치하면_유효성_검사가_성공한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/common/validation/FieldMatchValidatorTest.java:34]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '서로_일치하면_유효성_검사가_성공한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/test/java/com/patientpal/backend/common/validation/FieldMatchValidatorTest.java [hunk 23]
```diff
@@ -40,8 +43,7 @@
     }
 
     @Test
-    @DisplayName("서로 일치하지 않으면 유효성 검사가 실패한다.")
-    void fail() {
+    void 서로_일치하지_않으면_유효성_검사가_실패한다() {
```
Comment (su6net) [inline src/test/java/com/patientpal/backend/common/validation/FieldMatchValidatorTest.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Method name '서로_일치하지_않으면_유효성_검사가_실패한다' must match pattern '^[a-z][a-z0-9]\w*$'.

File: src/main/java/com/patientpal/backend/auth/controller/AuthenticationApiV1Controller.java [hunk 24]
```diff
@@ -1,41 +1,77 @@
...
     private final JwtTokenService tokenService;
+    private final JwtTokenProvider jwtTokenProvider;
     private final MemberService memberService;
 
     @PostMapping("/login")
-    public ResponseEntity<TokenResponse> authorize(@Valid @RequestBody SignInRequest request) {
-        return ResponseEntity.ok().body(loginService.authenticateUser(request));
+    @Operation(summary = "로그인", description = "로그인을 한다.")
+    @ApiResponse(responseCode = "200", description = "로그인 성공",
+        content = @Content(schema = @Schema(implementation = AccessTokenResponse.class)),
```
Comment (su6net) [inline src/main/java/com/patientpal/backend/auth/controller/AuthenticationApiV1Controller.java:42]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'content' has incorrect indentation level 8, expected level should be 12.

File: src/main/java/com/patientpal/backend/auth/controller/AuthenticationApiV1Controller.java [hunk 25]
```diff
@@ -1,41 +1,77 @@
...
+        headers = @Header(name = "Set-Cookie", description = "리프레시 토큰 쿠키 설정", schema = @Schema(type = "string")))
```
Comment (su6net) [inline src/main/java/com/patientpal/backend/auth/controller/AuthenticationApiV1Controller.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'headers' has incorrect indentation level 8, expected level should be 12.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java (hunk 1 above)
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.TypeNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>타입 이름 '사용자가_로그인_시에'은 패턴 '^[A-Z][a-zA-Z0-9]*$'을 따라야 합니다.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java (hunk 4 above)
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.TypeNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>타입 이름 '사용자가_리프레시_토큰_재발급_시에'은 패턴 '^[A-Z][a-zA-Z0-9]*$'을 따라야 합니다.

File: src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java (hunk 7 above)
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/controller/AuthenticationApiV1ControllerTest.java:112]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.TypeNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>타입 이름 '사용자가_회원가입_시에'은 패턴 '^[A-Z][a-zA-Z0-9]*$'을 따라야 합니다.

File: src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java (hunk 12 above)
Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:112]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'DB에_리프레시_토큰이_존재하지_않는_경우_검증에_실패한다' must contain no more than '1' consecutive capital letters.

Comment (su6net) [inline src/test/java/com/patientpal/backend/auth/service/RefreshTokenServiceTest.java:112]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>이름 'DB에_리프레시_토큰이_존재하지_않는_경우_검증에_실패한다'에는 연속된 대문자가 '1'개를 넘지 않아야 합니다.

File: src/main/java/com/patientpal/backend/auth/dto/TokenDto.java [hunk 26]
```diff
@@ -0,0 +1,4 @@
+package com.patientpal.backend.auth.dto;
+
+public record TokenDto(String accessToken, String refreshToken) {
```
Comment (LeeHusung) [inline src/main/java/com/patientpal/backend/auth/dto/TokenDto.java:3]: 혹시 TokenResponse에서 TokenDto로 클래스명을 변경하신 이유를 알 수 있을까요 ❓

**Review by LeeHusung — APPROVED** [review]
HttpOnly 로 인해 보안상 한층 더 업그레이드 된 것 같습니다 !
고생하셨습니다 ~ 👍

File: src/main/java/com/patientpal/backend/auth/dto/TokenDto.java (hunk 26 above)
Comment (su6net) [reply src/main/java/com/patientpal/backend/auth/dto/TokenDto.java:3]: @LeeHusung 응답으로 내보내다가 `refresh_token`은 바디로 내보내면 안된다는 걸 알게 되고, DTO는 Response는 아니지만 여전히 DTO임을 강조해두고 싶어서 `TokenDto`라고 붙였던 것 같습니다...!

**Pull request merged**